libs/canfund/src/api/cmc.rs

```diff
@@ -0,0 +1,162 @@
+use async_trait::async_trait;
```

Comment:
There's no dfinity crate with the logic of this file already right? if not, we could maybe put this on it's own lib and publish later as i imagine it would be helpful for other rust canisters.

Reply:
Shouldn't that go to the nns monorepo?